modules/__tests__/findUnreachableRoutes-test.js

```diff
@@ -0,0 +1,81 @@
+import expect from 'expect'
+import React from 'react'
+import Route from '../Route'
+import IndexRoute from '../IndexRoute'
+import { createRoutes } from '../RouteUtils'
+import createTransitionManager from '../createTransitionManager'
+import history from 'history'
```

Comment:
You should be importing `createMemoryHistory` and creating a `history` instance for your transition managers below.